### PR TITLE
Add list in document

### PR DESCRIPTION
**Pull request by stef74**
#1614

### CODE REVIEW

**Comment by stef74** [comment]
need to fix indentation

**Comment by stef74** [comment]
capture d ecran 2017-05-04 a 21 29 58

The layout of waypoint are adressed in #1612

**Comment by asaunier** [comment]
I think it's better to use the approach of PR https://github.com/c2corg/v6_ui/pull/1621

**Comment by stef74** [comment]
rebased and tested

File: less/lists.less [hunk 1]
```diff
@@ -450,9 +450,7 @@
   font-size: 0.9em;
 }
 
-
-.documents-list-section-container {
-  &.row-list {
+.row-list {
     a {
```
Comment (asaunier) [inline less/lists.less]: Indent is not correct around here

File: less/lists.less [hunk 2]
```diff
@@ -450,107 +450,50 @@
...
+    padding: 0px;
+  }
+  flex-direction: row;
+  .list-item {
+    flex-direction: row;
+    flex: 1 1 100%;
+    margin-top: 3px;
     a {
-      padding: 0px;
+     padding: 0px;
```
Comment (asaunier) [inline less/lists.less]: Indentation = 2 blank spaces

File: less/lists.less [hunk 3]
```diff
@@ -450,107 +450,50 @@
...
-              align-self: center;
-              padding-left: 2px;
-              padding-right: 2px;
-            }
-            .elevation {
-              width: 65px;
-            }       
+          .day {
+	    position: unset; 
+            background-color: unset;
```
Comment (asaunier) [inline less/lists.less]: Indentation = 2 blank spaces

File: less/lists.less [hunk 4]
```diff
@@ -566,30 +509,84 @@
...
             flex-shrink: 0;
-            align-self: center;
-            padding-left: 2px;
-            padding-right: 2px;
-          }
-          .elevation, .height-diff, .height-diff-difficulties {
+              align-self: center;
+              padding-left: 2px;
+              padding-right: 2px;
+            }
```
Comment (asaunier) [inline less/lists.less]: Why adding indentation here?

File: less/lists.less [hunk 5]
```diff
@@ -566,30 +509,84 @@
...
+          }       
+        }
+      }
+      .activity-author {
+        width: 30%;
+        align-self: center;		
+        .condition-rating {
+          order: 0;
+	  margin-left: 7%;
+	  margin-right: 7%;
```
Comment (asaunier) [inline less/lists.less]: Indentation again

**Comment by stef74** [comment]
rebased and fix little issue

File: less/lists.less [hunk 6]
```diff
@@ -451,172 +451,159 @@
...
+    flex-direction: row;
+    flex: 1 1 100%;
+    margin-top: 3px;
     a {
       padding: 0px;
     }
+  }
+  .list-item-info {
+     align-self: center;
+     flex-direction: row;
```
Comment (asaunier) [inline less/lists.less]: 3-space indentation instead of 2

**Pull request merged**